src/mozanalysis/metrics/desktop.py

```diff
@@ -11,18 +11,22 @@
     from_expr="`moz-fx-data-shared-prod.telemetry.clients_daily`",
 )
 
-#: DataSource: The `search_clients_daily`_ table.
+#: DataSource: The `search_clients_engines_sources_daily`_ table.
 #: This table unpacks search counts from the main ping;
 #: it contains one row per (client_id, submission_date, engine, source).
 #:
-#: .. _`search_clients_daily`: https://docs.telemetry.mozilla.org/datasets/
-#:    search/search_clients_daily/reference.html
-search_clients_daily = DataSource(
-    name="search_clients_daily",
-    from_expr="`moz-fx-data-shared-prod.search.search_clients_daily`",
+#: .. _`search_clients_engines_sources_daily`: https://docs.telemetry.mozilla.org/
+#:    datasets/search/search_clients_engines_sources_daily/reference.html
+search_clients_engines_sources_daily = DataSource(
+    name="search_clients_engines_sources_daily",
+    from_expr="`moz-fx-data-shared-prod.search.search_clients_engines_sources_daily`",
```

Comment:
Note that all user-facing views are published in both shared-prod and mozdata. Preferred style today would be to use mozdata and drop the backticks here.

Reply:
Thanks for the callout!